examples/meter/meter.js

```diff
@@ -0,0 +1,32 @@
+/** @type {!Element} Meter element. */
+let meterEl;
+/** @type {!Element} Element visually filling the meter. */
+let visualFillEl;
+/** @type {number} Minimum value. */
+let minUsage;
+/** @type {number} Maximum value. */
+let maxUsage;
+/** @type {number} Current value. */
+let usage;
+
+/** Set the value randomly between the minimum and maximum. */
+function randomizeUsage() {
+  const range = maxUsage - minUsage;
+  usage = Math.floor((Math.random() * range) + minUsage);
+  meterEl.setAttribute('aria-valuenow', usage);
+  render();
+}
+function render() {
+  const asPercentage = (100 * usage) / (maxUsage - minUsage);
+  visualFillEl.style.width = asPercentage + '%';
+}
+function init() {
```

Comment:
Since these are in the global namespace, would it make sense to define them as `initMeter`, `renderMeter`, and `randomizeMeterUsage`, just to be slightly safer? Another option would be to define everything within a `Meter` constructor function.

Reply:
> Another option would be to define everything within a `Meter` constructor function.

I'd favor this since it will make the migration to ES6 classes easier down the road.